src/airpyllution/airpyllution.py

```diff
@@ -243,11 +241,13 @@ def get_pollution_forecast(lat, lon, api_key):
     try:
         response = requests.get(url=url, params=params)
         response_obj = response.json()
-        data = convert_data_to_pandas(response_obj)
+        try:
+            data = convert_data_to_pandas(response_obj)
+        except:
+            if "cod" in response_obj:
+                return response_obj["message"]
     except:
-        if "cod" in response_obj:
```

Comment:
@christopheralex was just going through this code and realised this line would error here as "response_obj" would be undefined (because response_obj is only defined in the `try`). I've refactored it to look like the above two functions

Reply:
Not sure how it passed all the tests thou . Do we have a test for checking invalid API ?

Reply:
Oh I get what you saying . Incase the code errors out at `requests.get()` then response_obj isnt defined to be used in except block . Ya makes sense. Good observation Mel :)

Reply:
@mel-liow just make sure you run the tests again just for safety .

Reply:
Yeah it was passing because we were giving it a response object, so technically response_obj was fine. Your old code would have errored if `response` was some error (not an API error), and `response_obj` couldn't parse it as a json. 

Just writing a test for invalid api now!